### PR TITLE
Replacing Jquery with Fetch 

### PR DESCRIPTION
**Pull request by astha01**

### CODE REVIEW

**Review by Shraddha2104 — COMMENTED** [review]
I think you can remove the userInfo.html file from the commit since it's not being changed.

File: userinfo.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-     
+    
     <title>User Info</title>
 </head>
 <body>
```
Comment (apoorva-mk) [inline userinfo.html:8]: Hi, please resolve the conflicts. This file has been deleted.

File: public/scripts/main.js [hunk 2]
```diff
@@ -23,17 +23,19 @@ let questionStartMS = 0;
...
     }
-    // Call readText()
-    readText()
-    
-    updateTimingDisplay();
-  })
+     readText()
+     updateTimingDisplay()
+  }
+     );
```
Comment (apoorva-mk) [inline public/scripts/main.js:38]: Indenting of closing brace

File: public/scripts/main.js [hunk 3]
```diff
@@ -23,17 +23,19 @@ let questionStartMS = 0;
...
   //$("#retryButton").toggleClass("on");
   //alert($('li[data-keycode="test"]').attr('id'));
-  $.getJSON( "scripts/shortcuts.json", function( data ) {
-    allData = data;
+  
+  fetch('scripts/shortcuts.json')
+  .then(response => response.json())
+  .then(data => {
+    allData=data
     if(localStorage.getItem("questionNo")==null){
```
Comment (apoorva-mk) [inline public/scripts/main.js:31]: Could use a space here.

Comment (astha01) [reply public/scripts/main.js:31]: sure

**Review by astha01 — COMMENTED** [review]
Issue 95:  For our API calls in main.js we're currently using jQuery's AJAX functions. These are outdated and use non-standard promises.

Let's migrate these to use fetch

**Review by apoorva-mk — COMMENTED** [review]
There are still conflicts, looks like there's already been a PR merged addressing this issue :/

**Review by vegetabill — APPROVED** [review]
Looks great!

**Comment by vegetabill** [comment]
Sorry @astha01 someone didn't follow the process and already merged a different change that was the same. I'm going to have to close this PR.

**Comment by alodahl** [comment]
@vegetabill which change was merged in?

**Comment by apoorva-mk** [comment]
@alodahl another PR which didn't reference the issue has already been merged. Which is also why there were conflicts.

**Comment by alodahl** [comment]
Oh, I see.  https://github.com/Techtonica/keyboard-shortcuts-practice/pull/109/files

Sorry @astha01, here are some open issues: https://github.com/Techtonica/keyboard-shortcuts-practice/projects/4#column-4132662

**Comment by vegetabill** [comment]
@astha01 just letting you know we ended up merging this PR. Sorry for the confusion/miscommunication! Thanks for contributing.

**Pull request merged**